packages/editor/src/components/autocompleters/block.js

```diff
@@ -58,7 +58,17 @@ export function createBlockCompleter( {
 	return {
 		name: 'blocks',
 		className: 'editor-autocompleters__block',
-		triggerPrefix: '/',
```

Comment:
The power of the new `test` and `getQuery` functions is cool, but I'm wondering if it would be good to continue supporting a `trigger` property for people getting started or for those who don't need the power.

Reply:
Ok, let's leave the API alone until we work on the formatting API in the next PR. We should just focus on replacing the content change detection and manipulation logic.